service/src/test/resources/ehr/mapper/documentreference/expected-output-narrative-statement-with-nopat-15.xml

```diff
@@ -0,0 +1,26 @@
+<component typeCode="COMP" >
+    <NarrativeStatement classCode="OBS" moodCode="EVN">
+        <id root="394559384658936" />
+        <text>Type: Referral Letter (21-Sep-2020) Author Org: Organisation name Custodian Org: Organisation name Description: for 2 week rule referral - skin ${ERROR_MESSAGE_PLACEHOLDER_ID=394559384658936}Setting: Practice Setting Text</text>
+        <statusCode code="COMPLETE" />
+        <availabilityTime value="20200921" />
+        <confidentialityCode
+              code="NOPAT"
+              codeSystem="2.16.840.1.113883.4.642.3.47"
+              displayName="no disclosure to patient, family or caregivers without attending provider's authorization" />
```

Comment:
This needs to be within the `NarrativeStatement / reference / referredToExternalDocument`